MekHQ/src/mekhq/gui/dialog/MedicalViewDialog.java

```diff
@@ -499,30 +497,30 @@ private JPanel fillInjuries(JPanel panel, Campaign c, Person p) {
                         injLabel = genWrittenText(String.format(resourceMap.getString("injuriesTextAndDuration.format"), //$NON-NLS-1$
                             inj.getType().getSimpleName(), inj.getStart().toString(DATE_FORMATTER), genTimePeriod(inj.getTime())));
                     }
-                    if(isGMMode()) {
+                    if(campaign.isGM()) {
```

Comment:
spacing

Reply:
This is at the appropriate indentation level.